### PR TITLE
general edits

### PR DESCRIPTION
**Pull request by PeterZFarkas**

### CODE REVIEW

File: docs/source/examples/om6_wing_webUI.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ The mesh used for this case contains 113K nodes and 663K tetrahedrons, and the f
 
 - Mach Number = 0.84
 - Reynolds Number (based on MAC) = 11.72 Million
-- Alpha = 3.06 |deg|.
+- Alpha = 3.06 |deg|
```
Comment (cjdoolittle) [inline docs/source/examples/om6_wing_webUI.rst:22]: CJ adding comment

File: docs/source/preprocessing/manualMeshing.rst [hunk 2]
```diff
@@ -165,8 +165,8 @@ Flow360
 
 -   CGNS considerations:
 
-    -   CGNS mesh should be exported as an HDF5 file type
-    -   Boundaries **must** be exported as “Elements_t” type, which contains connectivity information necessary in Flow360
+    -   CGNS mesh should be exported as an `HDF5 <https://en.wikipedia.org/wiki/Hierarchical_Data_Format#HDF5>`_ file type
+    -   Boundaries **must** be exported as `“Elements_t” <https://cgns.github.io/CGNS_docs_current/sids/gridflow.html>`_ type, which contains connectivity information necessary in Flow360
```
Comment (will-flex) [inline docs/source/preprocessing/manualMeshing.rst:169]: Recommend adding #Elements to link: https://cgns.github.io/CGNS_docs_current/sids/gridflow.html#Elements

Directs user straight to Elements_t definition.

**Review by will-flex — CHANGES_REQUESTED** [review]

**Review by will-flex — APPROVED** [review]

**Review by will-flex — APPROVED** [review]

**Review by will-flex — DISMISSED** [review]

**Review by will-flex — DISMISSED** [review]

File: docs/source/examples/om6_wing_webUI.rst (hunk 1 above)
Comment (cjdoolittle) [reply docs/source/examples/om6_wing_webUI.rst:24]: fix in sec. 1.2 also

File: docs/source/preprocessing/manualMeshing.rst (hunk 2 above)
Comment (cjdoolittle) [reply docs/source/preprocessing/manualMeshing.rst:169]: agreed, add #Elements. Let's remove the HDF5 Wikipedia link, however.

File: docs/source/examples/om6_wing_webUI.rst [hunk 3]
```diff
@@ -8,8 +8,10 @@ Run CFD using Web UI: An example of ONERA M6 Wing
 The Onera M6 wing is a classic CFD validation case for external flows because of its simple geometry combined with complexities of transonic flow (i.e. local supersonic flow, shocks, and turbulent boundary layer separation).  It is a swept, semi-span wing with no twist and uses a symmetric airfoil using the ONERA D section.  More information about the geometry can be found at `NASA's website <https://www.grc.nasa.gov/www/wind/valid/m6wing/m6wing.html>`_.  The geometry parameters are:
```
Comment (cjdoolittle) [inline docs/source/examples/om6_wing_webUI.rst:8]: "The geometry parameters are" > "The geometry parameters of the model are"

File: docs/source/examples/om6_wing_webUI.rst [hunk 4]
```diff
@@ -8,8 +8,10 @@ Run CFD using Web UI: An example of ONERA M6 Wing
...
 
 - Mean Aerodynamic Chord (MAC) = 0.80167
-- Semi-span = 1.47602
-- Reference area = 1.15315
+- Semi-span = 1.47602 m
+- Reference area = 1.15315 m^2
```
Comment (cjdoolittle) [inline docs/source/examples/om6_wing_webUI.rst:12]: format to superscript

File: docs/source/examples/om6_wing_pyAPI.rst [hunk 5]
```diff
@@ -8,8 +8,8 @@ Run CFD using Python API: An example of ONERA M6 Wing
 The Onera M6 wing is a classic CFD validation case for external flows because of its simple geometry combined with complexities of transonic flow (i.e. local supersonic flow, shocks, and turbulent boundary layer separation).  It is a swept, semi-span wing with no twist and uses a symmetric airfoil using the ONERA D section.  More information about the geometry can be found at `NASA's website <https://www.grc.nasa.gov/www/wind/valid/m6wing/m6wing.html>`_.  The geometry parameters are:
```
Comment (cjdoolittle) [inline docs/source/examples/om6_wing_pyAPI.rst:8]: apply same changes as in om6_wing_webUI.rst here. You'll see in my recent commit that there are a number of text edits throughout the page. Please try to apply these changes where applicable.

File: docs/source/examples/om6_wing_webUI.rst [hunk 6]
```diff
@@ -19,7 +21,7 @@ The mesh used for this case contains 113K nodes and 663K tetrahedrons, and the f
```
Comment (cjdoolittle) [inline docs/source/examples/om6_wing_webUI.rst:21]: "113K nodes and 663K" > "113k nodes and 663k"

File: docs/source/preprocessing/automatedMeshing.rst [hunk 7]
```diff
@@ -40,7 +40,7 @@ which will create a sphere of radius = 1 at (0, 0, 0). The face of the sphere wi
 Surface Meshing
 ===============
 
-The surface mesher takes the geometry file and configuration JSON file as input parameters and generates a surface mesh. The meshing configuration file (or dict in python), also called surfaceMesh.json, contains information such as maximum element edge length, curvature resolution angle or growth rate of 2D layers. See JSON example:
+The surface mesher takes the geometry file and configuration `JSON <https://en.wikipedia.org/wiki/JSON>`_ file as input parameters and generates a surface mesh. The meshing configuration file (or dict in python), also called surfaceMesh.json, contains information such as maximum element edge length, curvature resolution angle or growth rate of 2D layers. See JSON example:
```
Comment (cjdoolittle) [inline docs/source/preprocessing/automatedMeshing.rst:43]: revert change

File: docs/source/solverConfiguration/betTable.csv [hunk 8]
```diff
@@ -1,7 +1,8 @@
 Options@ Default@ Description
 rotationDirectionRule@ "rightHand"@ [string] the rule for rotation direction and thrust direction, "rightHand" or "leftHand". A detailed explanation and some examples are shown at :ref:`bet_input`.
 centerOfRotation@ Empty@ [3-array] center of the Blade Element Theory (BET) disk
-axisOfRotation@ Empty@ [3-array] rotational axis of the BET disk
+axisOfRotation@ Empty@ [3-array] rotational axis of the BET disk, i.e. (+) thrust axis
+volumeName@ Empty@ [string] Name of the mesh-block in which the BET-Disk is located. If unspecified, default will be the highest-level mesh-block. Used for specifying a BET-disk within a moving reference frame.
```
Comment (cjdoolittle) [inline docs/source/solverConfiguration/betTable.csv:5]: I think that volumeName is only available in a beta-release, let's remove for now. If someone asked you to include this, then disregard and keep it in.

File: docs/source/solverConfiguration/solverConfiguration.rst [hunk 9]
```diff
@@ -272,6 +272,8 @@ turbulenceModelSolver
    +--------------------------------+-----------------+-------------------------------------------------------------------------------------------------------------------------------------------+
    | rotationCorrection             |      FALSE      | Rotation correction for the turbulence model. Only support for SpalartAllmaras                                                            |
    +--------------------------------+-----------------+-------------------------------------------------------------------------------------------------------------------------------------------+
+   | quadraticConstitutiveRelation  |      FALSE      | Use quadratic constitutive relation for turbulence shear stress tensor instead of Boussinesq Approximation                                |
```
Comment (cjdoolittle) [inline docs/source/solverConfiguration/solverConfiguration.rst:275]: I think that volumeName is only available in a beta-release, let's remove for now. If someone asked you to include this, then disregard and keep it in.

File: docs/source/capabilities/bladeElementTheory.rst [hunk 10]
```diff
@@ -213,6 +213,10 @@ where the moment center is the :code:`centerOfRotation` of each disk, defined in
 
    The above Force and Moment values mean the force and moment acted on **solid**. If you want to know the force and moment acted on **fluid**, just add a negative sign in front of it. 
 
+.. attention::
+
+   The x-, y-, z-component of Disk[diskID]_Force and Disk[diskID]_Moment is based on the inertial frame of reference, which is defined in the mesh file. The inertial frame of reference is fixed at the ground.
```
Comment (cjdoolittle) [inline docs/source/capabilities/bladeElementTheory.rst:218]: I don't fully understand this comment, can we revise?

**Review by cjdoolittle — CHANGES_REQUESTED** [review]

**Review by cjdoolittle — APPROVED** [review]

**Pull request merged**